hack/generator/pkg/astmodel/armconversion/convert_from_arm_function_builder.go

```diff
@@ -218,6 +219,23 @@ func (builder *convertFromARMBuilder) ownerPropertyHandler(
 	return []dst.Stmt{result}
 }
 
+// conditionsPropertyHandler generates conversions the "Conditions" status property. This property is set by the controller
+// after each reconcile and so currently is not preserved.
```

Comment:
"currently" implies that maybe it should in future? Is this right?
```suggestion
// after each reconcile and does not need to be preserved.
```

Reply:
There's a theoretical future where we have support for more than just the `Ready` condition, and other conditions aren't set in each reconcile loop pass possibly?

It seems an unlikely future though, especially because if we don't rederive the conditions each pass there are cases (such as when `Status` gets lost) where we might be making assumptions about what we have and not updating them - have made the change you suggested as it more clearly explains the _now_, and we can leave the future for the future.